### PR TITLE
Data models

### PR DESCRIPTION
**Pull request by rae-stanton**
Created models for user, recipe, and ingredient objects for use later.

### CODE REVIEW

**Comment by rae-stanton** [comment]
@spaulsandhu

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -3,8 +3,7 @@
   <nav class="navbar navbar-expand-lg navbar-light bg-white fixed-top">
     <div class="container-fluid">
     <a class="navbar-brand" href="#">
-      <img src="" alt="" width="30" height="24" class="d-inline-block align-text-top">
-      Recipe App
+      <%= image_tag 'hklogocircle.png', size: "75x75", class: "d-inline-block align-text-top rounded-circle"%>
```
Comment (spaulsandhu) [inline app/views/layouts/_navbar.html.erb:6]: This fine but it feels like it should be in its own PR. Just for future reference.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```
Comment (spaulsandhu) [inline app/models/user.rb:1]: This looks fine, I take it we'll be adding validations at a later date?

File: app/models/recipe.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Recipe < ApplicationRecord
+  belongs_to :user #author
```
Comment (spaulsandhu) [inline app/models/recipe.rb:2]: lets try `belongs_to :author, class_name: 'User'` and see if that works: https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/has_many

I think what we've discussed is allowing the interface to be something like `recipe.author => user` right?

We can setup associations with other models with arbitrary names like

`belongs_to :whatever_model` but we can manually point to model we'll be storing our information in. 

So an "Author" is a "kind of" "User" and we'd represent that in our ActiveRecord relation with:

`belongs_to :author, class_name: 'User`.

Try that change and then play with the models in the console like so:

```
1. rails c
2. user = User.create(name: 'Faust', email: 'faustbubba@bubba.boi')
3. recipe = user.recipes.create
4. recipe.author.name # => should return 'Faust'
```

**Comment by spaulsandhu** [comment]
This is a good start, take a look at some of the comments I've left and play around in the console after making a few of the suggested changes. We can work on this together after you've spend a bit of time trying this solo first. Let me know if you run into any issues and I'll be happy to pair with you.

File: app/models/recipe.rb (hunk 3 above)
Comment (spaulsandhu) [reply app/models/recipe.rb:2]: Take a look at this blog post and see if you can get `recipe.author` to work.

https://medium.com/@lushiyun/using-aliasing-associations-for-intuitive-development-reflections-from-a-rails-project-faea6c66f9af

**Pull request merged**